include/El/core/imports/choice.hpp

```diff
@@ -86,34 +86,35 @@ class Args
     vector<bool> usedArgs_;
     ostream& error_;
 
-    virtual void HandleVersion( ostream& os=cout ) const { }
-    virtual void HandleBuild( ostream& os=cout ) const { }
+    virtual void HandleVersion( ostream& os=cout ) const { (void)os; }
```

Comment:
p.s. one could as well had those `inline void Args::...` within the header below, the current diff is just smaller.

Reply:
I think defining an `EL_UNUSED` macro ala https://stackoverflow.com/a/1486931 would be more readable than the explicit voiding. Would you mind adding such a macro? This PR should be ready after that.

Reply:
Also, thanks!

Reply:
where do keep macros? where shall I put it?

Reply:
I think in include/El/core.hpp would be most consistent.